pyproject.toml

```
@@ -29,37 +29,37 @@ ignore_missing_imports = true
 
 [tool.poetry.dependencies]
 python = "^3.8.10"
-pydantic = "^2.7,<3.0"
-jsonschema = "^4.21.1"
-sentencepiece = "0.2.0"
-typing-extensions = "^4.11.0"
-tiktoken = "^0.7.0"
-pillow = "^10.3.0"
-requests = "^2.0.0"
+pydantic = ">=2.7,<3.0"
+jsonschema = ">=4.21.1"
+sentencepiece = ">=0.2.0"
+typing-extensions = ">=4.11.0"
+tiktoken = ">=0.7.0"
+pillow = ">=10.3.0"
+requests = ">=2.0.0"
 
 numpy = [
   { version = ">=1.22,<1.25", python = "<3.9" },
   { version = ">=1.25", python = ">=3.9" },
 ]
 
-opencv-python-headless = { version = "^4.0.0", optional = true }
+opencv-python-headless = { version = ">=4.0.0", optional = true }
 
 [tool.poetry.extras]
 opencv = ["opencv-python-headless"]
 
 [tool.poetry.group.dev.dependencies]
-types-jsonschema = "4.21.0.20240118"
-types-protobuf = "4.24.0.20240129"
-mypy-protobuf = "^3.5.0"
-pytest = "7.4.4"
+types-jsonschema = ">=4.21.0.20240118"
+types-protobuf = ">=4.24.0.20240129"
+mypy-protobuf = ">=3.5.0"
+pytest = ">=7.4.4"
 ruff = "^0.2.2"
-mypy = "^1.8.0"
-pytest-cov = "^4.1.0"
-diff-cover = "^8.0.3"
-types-Pillow = "^10.2.0"
-types-requests = "^2.32"
+mypy = ">=1.8.0"
+pytest-cov = ">=4.1.0"
+diff-cover = ">=8.0.3"
+types-Pillow = ">=10.2.0"
+types-requests = ">=2.32"
 
-coverage = { extras = ["toml"], version = "^7.4.4" }
+coverage = { extras = ["toml"], version = ">=7.4.4" }
```

```suggestion
coverage = { extras = ["toml"], version = ">=7.4.4" }
```

`toml` is already >= 10.0